src/main/java/org/breedinginsight/brapi/v2/BrAPIV2Controller.java

```diff
@@ -186,6 +187,15 @@ public HttpResponse<?> getCatchall(@PathVariable("path") String path, @PathVaria
         return executeRequest(path, programId, request, "GET");
     }
 
+    @Delete("/${micronaut.bi.api.version}/programs/{programId}" + BrapiVersion.BRAPI_V2 + "/{+path}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.PROGRAM_SCOPED_ROLES})
```

Comment:
Is this something only a program or system admin should be allowed to do? Wondering if we should make a new role group that is system admin and program admin without readonly.